atmo/coordinator/coordinator_websocket.go

```diff
@@ -0,0 +1,82 @@
+package coordinator
+
+import (
+	"net/http"
+
+	"github.com/google/uuid"
+	"github.com/gorilla/websocket"
+	"github.com/pkg/errors"
+	"github.com/suborbital/atmo/directive"
+	"github.com/suborbital/reactr/request"
+	"github.com/suborbital/vektor/vk"
+)
+
+func (c *Coordinator) websocketHandlerForDirectiveHandler(handler directive.Handler) http.HandlerFunc {
+	upgrader := websocket.Upgrader{} // use default options
```

Comment:
Should specific WebSocket upgrader options be exposed in Directive.yaml? (buffer sizes, timeouts, etc.)